nservicebus/upgrades/3to4.md

```diff
@@ -146,6 +146,7 @@ NServiceBus PowerShell cmdlets have moved to NServiceBus.PowerShell.dll. See [Ma
 
 `configure.JsonSerializer()` has been obsoleted, please use `Configure.Serialization.Json();` instead. More info on supported serializers [here](/nservicebus/serialization/).
 
+The `XmlSerializer` will now automatically escape outgoing messages containing invalid characters. Version 3 endpoints will not be able to process those messages correctly.
```

Comment:
@SimonCropp In v3 endpoints, messages with invalid characters will throw an exception during serializing/deserializing. It won't be able to read messages with encoded invalid chars from v4.

I assumed that since this is an upgrade guide between two specific versions, it's time-constrained, "will now" has a specific meaning here. It mentions what changed in this version vs the previous.

But I'm all for better text from a native English speaker :)